istio/tests/test_check.py

```diff
@@ -0,0 +1,135 @@
+# (C) Datadog, Inc. 2010-2017
```

Comment:
Can we extend a bit the test coverage, by testing separately the methods of the check class beside the overall collection logic?

Reply:
Sure I can do this